cas-client-core/src/main/java/org/jasig/cas/client/util/CommonUtils.java

```diff
@@ -438,8 +439,12 @@ public static String getResponseFromServer(final URL constructedUrl, final HttpU
             }
 
             return builder.toString();
+        } catch (final SSLException e) {
+            LOGGER.error("SSL error getting response from host: {} : Error Message: {}", constructedUrl.getHost(), e.getMessage(), e);
+            throw new RuntimeException(e);        	
         } catch (final Exception e) {
-            LOGGER.error(e.getMessage(), e);
+            LOGGER.error("Error getting response from host: {} with path: {} and protocol: {}: Error Message: {}",
+            		constructedUrl.getHost(), constructedUrl.getPath(), constructedUrl.getProtocol(), e.getMessage(), e);
```

Comment:
Are you referring to when people are using slf4j pre-1.6.0? As of 1.6.0, which came out in 2010, that behavior changed:  https://www.slf4j.org/faq.html#paramException

I can change it to use an Object array and try to test it later today if you are looking to support old sl4fj versions.